### PR TITLE
Add changelog_uri to gemspec

### PR DESCRIPTION
**Pull request by fynsta**
Supported here: https://guides.rubygems.org/specification-reference/#metadata Useful for running https://github.com/MaximeD/gem_updater

### CODE REVIEW

**Review by david942j — CHANGES_REQUESTED** [review]
Thanks for the contribution! I believe you have a typo that `spec` should be replaced by `s`.

File: elftools.gemspec [hunk 1]
```diff
@@ -18,6 +18,10 @@ Gem::Specification.new do |s|
   s.files       = Dir['lib/**/*.rb'] + %w[README.md]
   s.homepage    = 'https://github.com/david942j/rbelftools'
   s.license     = 'MIT'
+
+  s.metadata['rubygems_mfa_required'] = 'true'
+  s.metadata['changelog_uri'] = "#{spec.homepage}/releases/tag/v#{spec.version}",
```
[the remark this answers is not in the document]
Comment (fynsta) [reply elftools.gemspec:23]: Great catch, thanks!

File: elftools.gemspec [hunk 2]
```diff
@@ -18,6 +18,10 @@ Gem::Specification.new do |s|
   s.files       = Dir['lib/**/*.rb'] + %w[README.md]
   s.homepage    = 'https://github.com/david942j/rbelftools'
   s.license     = 'MIT'
+
+  s.metadata['rubygems_mfa_required'] = 'true'
+  s.metadata['changelog_uri'] = "#{s.homepage}/releases/tag/v#{s.version}",
```
Comment (david942j) [inline elftools.gemspec:23]: need to remove ',' at the end of this line

**Review by david942j — CHANGES_REQUESTED** [review]

File: elftools.gemspec (hunk 2 above)
Comment (fynsta) [reply elftools.gemspec:23]: Damn. So sorry for the added work, this was copied from a different contribution I made somewhere else.

**Comment by david942j** [comment]
Merged, thanks!

**Pull request merged**